src/app/stake/helpers.ts

```diff
@@ -203,14 +203,16 @@ export const createValidatorList = (
   chainsDetails: (GetChainDetailsResponse | undefined | null)[],
   mobulaMarketData: MobulaMarketMultiDataResponse | undefined | null
 ): Validator[] => {
-  return validatorData.reduce<Validator[]>((acc, current) => {
+  let validators = validatorData.reduce<Validator[]>((acc, current) => {
```

Comment:
`const` should always be preferred over `let`, there are often ways to write the code so that we don't need to modify a variable after its declaration (but in some cases it's too complicated and not worth it)

Reply:
Indeed, here you don't need a new variable, and you don't need to change anything in the `validatorsData.reduce(...)` instruction.
You just need to add your `.sort(...)` right after it:
```
return validatorData.reduce<Validator[]>(...).sort(...);
```

 Many native javascript functions can be chained that way :magic_wand: